File: Curso Programação Orientada a Objetos/Java com POO/Aula12e13/src/aula12e13/Canguru.java
package aula12e13;

public class Canguru extends Mamifero{
    //Métodos
    public void usarBolsa() {
        System.out.println("Canguru está usando sua bolsa!");
    }
    
    //Métodos Sobrepostos
    @Override
    public void locomover() {
        System.out.println("Saltando");
    }
}

File: Curso Programação Orientada a Objetos/PHP com POO/Aula12/Canguru.php
<?php

require_once 'Mamifero.php';

class Canguru extends Mamifero {
    //Métodos
    public function usarBolsa() {
        echo "<p>Canguru está usando sua bolsa!</p>";
    }
    
    //Métodos Sobrepostos
    public function locomover() {
        echo "<p>Saltando</p>";
    }
}
